README.md

```diff
@@ -53,6 +53,22 @@ tcld namespace update accepted-client-ca set -n <namespace> --ca-certificate-fil
 ```
 > :warning: If the update removes a certificate, any clients (tctl/workers) still using the removed certificate will fail to connect to the namespace after the update completes.
 
+#### Performing a Certificate Rollover:
+It's important to do a rollover process when updating your CA certificates. This allows your namespace to serve both CA certs for a period of time until traffic to your old cert is gone. To do this follow these steps:
```

Comment:
In the interest of global English, I recommend spelling out "cert" globally (`s/cert/certificate/`). If not, shorten "certificate" in step 3 for consistency.